cpp/ql/test/library-tests/dataflow/DefaultTaintTracking/annotate_sinks_only/tainted.expected

```diff
@@ -0,0 +1,2 @@
+| defaulttainttracking.cpp:190:14:190:24 | // $ ast,ir | Missing result:ir= |
+| defaulttainttracking.cpp:193:14:193:24 | // $ ast,ir | Missing result:ir= |
```

Comment:
As @MathiasVP remarked in https://github.com/github/codeql/pull/5509#discussion_r600845166, this file should be empty.